tests/acceptance/tests/Settings/WishlistGuestFunctionality.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test } from '@fixtures/AcceptanceTest';
+
+test('Customer is able to add and remove products to the wishlist',{ tag: '@Wishlist' }, async ({ 
+    TestDataService,
+    ShopCustomer,
+    StorefrontHome,
+    AddProductToWishlist,
+    StorefrontWishlist,
+    AddProductToCartFromWishlist,
+    Login
+}) => {
+    await TestDataService.setSystemConfig({ 'core.cart.wishlistEnabled': true });
+    const product1 = await TestDataService.createBasicProduct();
+    const product2 = await TestDataService.createBasicProduct();
+    const product1Locators = await StorefrontHome.getListingItemByProductId(product1.id);
+    const product2Locators = await StorefrontHome.getListingItemByProductId(product2.id);
+
+    await test.step('Add to products to the wishlist and check the basket count', async () => {
+        await TestDataService.setSystemConfig({ 'core.basicInformation.acceptAllCookies': true });
+        await ShopCustomer.goesTo(StorefrontHome.url());
+        await StorefrontHome.consentAcceptAllCookiesButton.click();
+        await StorefrontHome.page.reload();
+    });
+
+    await test.step('Add two products to the wishlist', async () => {
```

Comment:
You added only first product to the wishlist
```suggestion
    await test.step('Add product1 to the wishlist and verify wishlist count', async () => {
```

Reply:
Should be fixed 👍